go.mod

```diff
@@ -4,25 +4,26 @@ go 1.16
 
 require (
 	decred.org/cspp v0.3.0
-	github.com/decred/dcrd/addrmgr/v2 v2.0.0-20210901152745-8830d9c9cdba
-	github.com/decred/dcrd/blockchain/stake/v4 v4.0.0-20210901152745-8830d9c9cdba
-	github.com/decred/dcrd/blockchain/standalone/v2 v2.0.1-0.20210901152745-8830d9c9cdba
-	github.com/decred/dcrd/blockchain/v4 v4.0.0-20210901152745-8830d9c9cdba
-	github.com/decred/dcrd/certgen v1.1.2-0.20210901152745-8830d9c9cdba
-	github.com/decred/dcrd/chaincfg/chainhash v1.0.3-0.20210901152745-8830d9c9cdba
-	github.com/decred/dcrd/chaincfg/v3 v3.0.1-0.20210901152745-8830d9c9cdba
-	github.com/decred/dcrd/connmgr/v3 v3.0.1-0.20210901152745-8830d9c9cdba
-	github.com/decred/dcrd/crypto/blake256 v1.0.1-0.20210901152745-8830d9c9cdba
-	github.com/decred/dcrd/crypto/ripemd160 v1.0.2-0.20210901152745-8830d9c9cdba
-	github.com/decred/dcrd/dcrec v1.0.1-0.20210901152745-8830d9c9cdba
-	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.0.0-20210901152745-8830d9c9cdba
-	github.com/decred/dcrd/dcrjson/v3 v3.1.1-0.20210901152745-8830d9c9cdba
-	github.com/decred/dcrd/dcrutil/v4 v4.0.0-20210901152745-8830d9c9cdba
-	github.com/decred/dcrd/gcs/v3 v3.0.0-20210901152745-8830d9c9cdba
-	github.com/decred/dcrd/hdkeychain/v3 v3.0.1-0.20210901152745-8830d9c9cdba
-	github.com/decred/dcrd/rpc/jsonrpc/types/v3 v3.0.0-20210901152745-8830d9c9cdba
-	github.com/decred/dcrd/txscript/v4 v4.0.0-20210901152745-8830d9c9cdba
-	github.com/decred/dcrd/wire v1.4.1-0.20210901152745-8830d9c9cdba
+	github.com/decred/dcrd/addrmgr/v2 v2.0.0-20210914212651-723d86274b0d
+	github.com/decred/dcrd/blockchain/stake/v4 v4.0.0-20210914212651-723d86274b0d
+	github.com/decred/dcrd/blockchain/standalone/v2 v2.0.1-0.20210914212651-723d86274b0d
+	github.com/decred/dcrd/blockchain/v4 v4.0.0-20210914212651-723d86274b0d
+	github.com/decred/dcrd/certgen v1.1.2-0.20210914212651-723d86274b0d
+	github.com/decred/dcrd/chaincfg/chainhash v1.0.4-0.20210914212651-723d86274b0d
+	github.com/decred/dcrd/chaincfg/v3 v3.0.1-0.20210914212651-723d86274b0d
+	github.com/decred/dcrd/connmgr/v3 v3.0.1-0.20210914212651-723d86274b0d
+	github.com/decred/dcrd/crypto/blake256 v1.0.1-0.20210914212651-723d86274b0d
+	github.com/decred/dcrd/crypto/ripemd160 v1.0.2-0.20210914212651-723d86274b0d
+	github.com/decred/dcrd/database/v3 v3.0.0-20210914212651-723d86274b0d // indirect
+	github.com/decred/dcrd/dcrec v1.0.1-0.20210914212651-723d86274b0d
+	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.0.1-0.20210914212651-723d86274b0d
```

Comment:
There are no changes toward a secp256k1 v4.0.1 yet, so I suspect you want to require that actual v4.0.0 tag, just like with dcrjson/v4

Reply:
i have a script that just updates everything to the latest master. once we prepare for the release, all of these will be switched to using released versions.